src/app/message/message.service.ts

```diff
@@ -54,7 +54,7 @@ export class MessageService {
     });
   }
 
-  public synchronize(timeFrameStart: Date, timeFrameEnd: Date, countOnly = false): Observable<any> {
+  public synchronize(timeFrameStart: Date, timeFrameEnd: Date, countOnly: Boolean = false): Observable<any> {
```

Comment:
already merged, but primitive types need to be written in lowercase. Can you please fix this?